cdc/puller/sorter/unified_sorter.go

```diff
@@ -41,6 +41,8 @@ type UnifiedSorter struct {
 	dir         string
 	pool        *backEndPool
 	metricsInfo *metricsInfo
+
+	closeCh chan struct{}
```

Comment:
Could you add some comments to describe the exit workflow of a unified sorter and why cancelling Unified Sorter's Run does not cancel AddEntry, such as the hierarchy of context used in `Run` and `AddEntry`

Reply:
This is not a problem of the sorter itself, but of the caller(s). There is no guarantee that the contexts used to call `Run` and `AddEntry` are the same one.

Reply:
Yes, what I mean is there is no limit of the context hierarchy here, please add comments to address it